2023-03-15 14:25:01,230 INFO    MainThread:23612 [wandb_setup.py:_flush():76] Configure stats pid to 23612
2023-03-15 14:25:01,231 INFO    MainThread:23612 [wandb_setup.py:_flush():76] Loading settings from C:\Users\MIELEA1\.config\wandb\settings
2023-03-15 14:25:01,231 INFO    MainThread:23612 [wandb_setup.py:_flush():76] Loading settings from C:\Users\MIELEA1\Documents\TTJudge\wandb\settings
2023-03-15 14:25:01,231 INFO    MainThread:23612 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-15 14:25:01,231 INFO    MainThread:23612 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-03-15 14:25:01,231 INFO    MainThread:23612 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train_table_detection.py', 'program': 'c:\\Users\\MIELEA1\\Documents\\TTJudge\\train_table_detection.py'}
2023-03-15 14:25:01,231 INFO    MainThread:23612 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-03-15 14:25:01,232 INFO    MainThread:23612 [wandb_init.py:_log_setup():506] Logging user logs to C:\Users\MIELEA1\Documents\TTJudge\wandb\run-20230315_142501-056yr8tz\logs\debug.log
2023-03-15 14:25:01,232 INFO    MainThread:23612 [wandb_init.py:_log_setup():507] Logging internal logs to C:\Users\MIELEA1\Documents\TTJudge\wandb\run-20230315_142501-056yr8tz\logs\debug-internal.log
2023-03-15 14:25:01,232 INFO    MainThread:23612 [wandb_init.py:init():546] calling init triggers
2023-03-15 14:25:01,232 INFO    MainThread:23612 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {}
2023-03-15 14:25:01,232 INFO    MainThread:23612 [wandb_init.py:init():602] starting backend
2023-03-15 14:25:01,232 INFO    MainThread:23612 [wandb_init.py:init():606] setting up manager
2023-03-15 14:25:01,242 INFO    MainThread:23612 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-03-15 14:25:01,250 INFO    MainThread:23612 [wandb_init.py:init():613] backend started and connected
2023-03-15 14:25:01,255 INFO    MainThread:23612 [wandb_init.py:init():701] updated telemetry
2023-03-15 14:25:01,324 INFO    MainThread:23612 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-03-15 14:25:01,695 ERROR   MainThread:23612 [wandb_init.py:init():764] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-03-15 14:25:02,719 WARNING MsgRouterThr:23612 [router.py:message_loop():77] message_loop has been closed
